gradle.properties

```
@@ -0,0 +1,12 @@
+org.gradle.caching=true
+group=io.ballerina.lib
+
+# This is a dummy value as repo contains multiple packages
+version=1.0.0-SNAPSHOT
+qmVersion=1.0.0
```

Developement version will always be a SNAPSHOT,
```suggestion
qmVersion=1.0.0-SNAPSHOT
```